report/src/main/kotlin/com/hartwig/actin/report/pdf/tables/trial/EligibleExternalTrialsGenerator.kt

```diff
@@ -73,4 +76,58 @@ class EligibleExternalTrialsGenerator(
 
         return makeWrapping(table)
     }
+}
+
+fun provideExternalTrialsTablesIncludedByFilter(
```

Comment:
Normally we would use a companion object for these kind of factory methods, such that it looks more like we're used to with static methods on java classes.